### PR TITLE
Storybook

### PR DESCRIPTION
**Pull request by SaraVieira**
Adds an instance of storybook

http://striped-storybook-123.surge.sh

### CODE REVIEW

File: packages/common/storybook-static/iframe.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<!doctype html><html><head><meta charset="utf-8"><title>Storybook</title><meta name="viewport" content="width=device-width,initial-scale=1"><base target="_parent"><style>:not(.sb-show-main) > .sb-main,
```
Comment (libetl) [inline packages/common/storybook-static/iframe.html]: you should exclude storybook-static from git.

**Comment by libetl** [comment]
@ndelangen Look at this

File: packages/common/storybook-static/iframe.html (hunk 1 above)
Comment (SaraVieira) [reply packages/common/storybook-static/iframe.html]: 🤦🏻‍♀️

True! Thanks!

**Comment by SaraVieira** [comment]
https://codesandbox-storybook.now.sh

New Link

**Comment by SaraVieira** [comment]
Pending the merging of this PR on `gatsby-plugin-airtable`

https://github.com/jbolda/gatsby-source-airtable/pull/45

File: package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     "start:dynamic": "lerna run dev --scope dynamic-pages --stream",
     "start:common": "lerna run start --scope common --stream",
     "start:home": "lerna run start --scope homepage --stream",
+    "start:storybook": "lerna run storybook --scope common --stream",
```
Comment (siddharthkp) [inline package.json]:
```suggestion
    "start:storybook": "lerna run storybook --scope @codesandbox/common --stream",
```

Lerna expects the package name

File: package.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
     "start:dynamic": "lerna run dev --scope dynamic-pages --stream",
     "start:common": "lerna run start --scope common --stream",
```
Comment (siddharthkp) [inline package.json]:
```suggestion
    "start:common": "lerna run start --scope @codesandbox/common --stream",
```

Ditto here, package name

**Comment by siddharthkp** [comment]
Tried setting this up locally, the error I get is:

```
/codesandbox-client/node_modules/serve-favicon/index.js:55
    throw new TypeError('path to favicon.ico is required')
    ^

TypeError: path to favicon.ico is required
```

🤷‍♀ 

It's coming from [`storybook/core`](https://github.com/storybooks/storybook/blob/next/lib/core/src/server/build-dev.js). After some digging it looks storybook expects a `staticDir` in options which it couldn't figure out 🤔 

@SaraVieira Any clue here?

**Comment by SaraVieira** [comment]
Hey @siddharthkp 

So I updated all the yarns, and fixed all the conflicts and seems to have fixed the issue

I did find something I forgot to tell you: You need two panes open running:

```bash
yarn start:common
```

and

```
yarn start:storybook
```

Because storybook uses the built files
Ping me with any more issues

**Comment by SaraVieira** [comment]
Closed due to innactivity

**Pull request closed without merging**